tests/pytest/test_jkmemory.py

```diff
@@ -0,0 +1,51 @@
+"""
+Tests for the memory estimators on JK objects
+"""
+
+import psi4
+import pytest
+from utils import *
+
```

Comment:
add https://github.com/psi4/psi4/blob/master/tests/pytest/test_qcvars.py#L9 so it always runs in CI

Reply:
Not particularly quick, ok to run later?

Reply:
sure. no label means it runs in "full" (default) and "long" modes.